### PR TITLE
Added setup details to README

### PR DESCRIPTION
**Pull request by gsidebo**
#### What are the relevant tickets?
N/A

#### What's this PR do?
Adds some simple setup details to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # ocw-www
 
-## testimonials
+## Running the site locally
+
+#### Dependencies
+- If running for the first time, or if dependencies have changed, install dependencies: `yarn install`
+- This site requires a running instance of `ocw-studio` (the [repo](https://github.com/mitodl/ocw-studio) has
```
Comment (Ferdi) [inline README.md:7]: ocw studio should not be a dependency.   News and New courses depend on it only.  Ideally, if ocw studio is down, we'd simly not see those 2 carousels

Comment (gsidebo) [reply README.md:7]: As it stands, the site doesn't run if you don't specify a valid URL for a running ocw-studio instance. I can create an issue for running the site without those services

ss 2021-01-15 at 13 11 28

Comment (aliceriot) [reply README.md:7]: I think this is a good idea, the build shouldn't just blow up if you don't have ocw-studio running

**Review by aliceriot — APPROVED** [review]
:+1:

**Pull request merged**